### PR TITLE
Tears of Amaterasu - full spoiler

### PR DESCRIPTION
**Pull request by mplain**

### CODE REVIEW

File: Card/doji-representative.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "name": "Doji Representative",
+        "political": 3,
+        "political_bonus": null,
+        "role_restriction": null,
+        "side": "dynasty",
+        "strength": null,
+        "strength_bonus": null,
+        "text": "<b>Action:</b> During a conflict – move this character home.",
+        "traits": [
+            "courtiet"
```
Comment (Alsciende) [inline Card/doji-representative.json]: typo

File: Card/gifted-tactician.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "military": 2,
+        "military_bonus": null,
+        "name": "Gifted Tactician",
+        "political": 1,
+        "political_bonus": null,
+        "role_restriction": null,
+        "side": "dynasty",
+        "strength": null,
+        "strength_bonus": null,
+        "text": "<b>Reaction:</b> After this character wins a [military] conflict – draw 1 card.",
```
Comment (Alsciende) [inline Card/gifted-tactician.json]: [conflict-military]

File: Card/hito-district.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        "military": null,
+        "military_bonus": null,
+        "name": "Hito District",
+        "political": null,
+        "political_bonus": null,
+        "role_restriction": null,
+        "side": "dynasty",
+        "strength": null,
+        "strength_bonus": "+1",
+        "text": "Limit 1 per deck.\n[poitical] conflicts cannot be declared against this holding's province.",
```
Comment (Alsciende) [inline Card/hito-district.json]: typo + [conflict-political]

File: Card/raise-the-alarm.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        "military": null,
+        "military_bonus": null,
+        "name": "Raise the Alarm",
+        "political": null,
+        "political_bonus": null,
+        "role_restriction": null,
+        "side": "conflict",
+        "strength": null,
+        "strength_bonus": null,
+        "text": "<b>Action:</b> During a [military] conflict in which you are the defending player – turn the dynasty card in the attacked province faceup. Then, if that card is a character, put it into play as a defender.",
```
Comment (Alsciende) [inline Card/raise-the-alarm.json]: [conflict-military]

File: Card/seal-of-the-crane.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        "military": null,
+        "military_bonus": "+0",
+        "name": "Seal of the Crane",
+        "political": null,
+        "political_bonus": "+1",
+        "role_restriction": null,
+        "side": "conflict",
+        "strength": null,
+        "strength_bonus": null,
+        "text": "Attached character gains the [crane] clan symbol and the <em>Duelist</em> trait.",
```
Comment (Alsciende) [inline Card/seal-of-the-crane.json]: [clan-crane]

**Comment by mplain** [comment]
Fixed.

Too used to thronesdb format =_=

**Comment by Alsciende** [comment]
I'll add some CI checks, so that easy mistakes like that will be immediately reported.

**Pull request merged**